PencilCase/PencilCase/www/index.html

```diff
@@ -215,7 +215,7 @@ <h1><span data-bind="text: title"></span></h1>
             <div data-role="navbar" data-grid="a">
                 <ul>
                     <li>
-                        <a id="editButton" href="#" data-bind="click: startEditing, visible: !isInEditingMode()" data-icon="edit">Edit</a>
+                        <a id="editButton" href="#" data-bind="click: startEditing, visible: !isInEditingMode(), css: {'ui-btn-active': isInEditingMode}" data-icon="edit">Edit</a>
```

Comment:
This is one workaround for the following issue:
When editing one existed product, and then click back button, it can go back to the details mode, but on the UI, the "Edit" button is in active mode (class: ui-btn-active), expected result is that, no footer button is in active mode. 
So, I added this KO binding to remove the ui-btn-active class.

Issue: There is one issue with this workaround, when user click edit button to enter editing mode, at the click time point, the button won't in active mode(back ground in blue color and so on), you can refer to the operation to click save button in editing mode. Whatever, it's a tiny issue and I am not going to fix it.